### DISCUSSION TITLE
AWS AMI Install

### DISCUSSION THREAD

**Discussion by ob1xxx** [Security-Onion-Solutions/securityonion#5964] [Unsupported Versions] [answered]
I'm installing the 2.0 version from the AWS AMI. 2 issues. 

1. When entering a hostname, as I understand, this can be anything as it's not FQDN, I get errors at build. I'm naming it simply "master".
2. At build I'm getting these "salt-minion" errors. I don't understand what I'm to do here as this is a fresh install from the AMI via EC2 launch.

**Comment by jertel** [comment, marked as answer]
There was a yum repo issue yesterday that was later self-corrected. This could have affected the setup process. I suggest retrying and if you continue having an issue please provide a screenshot or log showing the errors.

**Comment by ob1xxx** [comment]
Thanks. The install is solid now, but I'm seeing another issue. I've confirmed traffic mirroring is functional via tcpdump, but there's no alerts for the target IP I'm attacking, nor is there any data from bro or zeek. This is happening on another instance I'm running in AWS with an older version of Security Onion. It's a bit frustrating as "so-test" has output, but nothing for the target IP.  On the older version I've switched the monitoring interface, but nothing. 

I'd like version 2-AMI to be my primary, but with no alerts, I'm confused as to how to fix this. Ideas?

**Reply by jertel** [reply]
This should be a new discussion. I suggest reviewing Zeek logs and If you still need help create a new discussion, since this one was originally related to the setup failing to complete.
